robottelo/cli/hostcollection.py

```diff
@@ -11,14 +11,14 @@
 
 Subcommands::
 
-    add-content-host              Add content host to the host collection
-    content-hosts                 List content hosts in the host collection
+    add-host              Add content host to the host collection
```

Comment:
Formatting here is also seem a little bit shifted or it is an issue of diffs? :)